opentelemetry-jaeger/README.md

```diff
@@ -41,9 +41,10 @@ exporting telemetry:
 ```rust
 use opentelemetry::global;
 use opentelemetry::trace::Tracer;
+use opentelemetry_jaeger_propagator;
```

Comment:
this readme file needs to updated to reflect it no longer has propagator and warning about exporter being removed in the next release.

Reply:
https://github.com/open-telemetry/opentelemetry-rust/pull/1487/files#diff-d153afbd7102b8218509b0ce0b654c51ba1b8fee4c558c097f01226743a89870R7 need a small fix to remove propagator wording